arch/power/adler32_vmx.c

```diff
@@ -0,0 +1,130 @@
+/* adler32_vmx.c -- compute the Adler-32 checksum of a data stream
+ * Copyright (C) 1995-2011 Mark Adler
+ * Copyright (C) 2017-2021 Mika T. Lindqvist <postmaster@raasu.org>
+ * For conditions of distribution and use, see copyright notice in zlib.h
+ */
+
+#ifdef PPC_VMX_ADLER32
+
+#include <altivec.h>
+#ifdef ZLIB_COMPAT
+#  include "zlib.h"
+#else
+#  include "zlib-ng.h"
+#endif
+#include "zutil.h"
+#include "adler32_p.h"
+
+#define vmx_zero()  (vec_splat_u32(0))
+
+vector unsigned short vec_hadduh(vector unsigned char a)
+{
```

Comment:
I think we have been using `{` that are on the same line as the function definition?

Reply:
Fixed.